services/venice-controller/src/main/java/com/linkedin/venice/controller/server/ClusterAdminOpsRequestHandler.java

```diff
@@ -108,11 +107,14 @@ public AdminTopicMetadataGrpcResponse getAdminTopicMetadata(AdminTopicMetadataGr
     return AdminTopicMetadataGrpcResponse.newBuilder().setMetadata(adminMetadataBuilder.build()).build();
   }
 
-  public UpdateAdminTopicMetadataGrpcResponse updateAdminTopicMetadata(UpdateAdminTopicMetadataGrpcRequest request) {
+  public AdminTopicMetadataGrpcResponse updateAdminTopicMetadata(UpdateAdminTopicMetadataGrpcRequest request) {
     AdminTopicGrpcMetadata metadata = request.getMetadata();
     String clusterName = metadata.getClusterName();
+    ControllerRequestParamValidator.validateAdminTopicMetadataRequest(clusterName);
+    if (!metadata.hasExecutionId()) {
+      throw new VeniceException("Execution id is required to update admin topic metadata");
```

Comment:
We should move `metadata.hasExecutionId()` check inside `ControllerRequestParamValidator.validateAdminTopicMetadataRequest()` and use `IllegalArgumentException` to make sure the proper error code is sent to the clients

Reply:
Or just use `ControllerRequestParamValidator.validateAdminCommandExecutionRequest(clusterName, executionId)`

Reply:
Take a closer look, we can use `ControllerRequestParamValidator.validateAdminCommandExecutionRequest(clusterName, executionId)` since if the executionId is not provided, the default value for getExecutionId will be 0, which will be caught in the validator.

Reply:
I have updated the code to use `ControllerRequestParamValidator.validateAdminCommandExecutionRequest(clusterName, executionId)`. Thanks for your suggestion.